projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/edit-user-profile/edit-user-profile.component.scss

```diff
@@ -0,0 +1,41 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
+.profile-dialog {
+  nb-card-header {
+    .navigation-buttons {
+      position: relative;
+      top: 17px;
```

Comment:
Any reason for these numbers?

Reply:
To display the blue underline of update basic details & email preferences on the nb-card header border bottom